### DISCUSSION TITLE
SRAF 崩坏：星穹铁道成就格式

### DISCUSSION THREAD

**Discussion by Lightczx** [UIGF-org/.github#9] [Ideas]
暂时还没有示例数据

**Comment by LaoshuBaby** [comment]
确实，巡回到这里发现居然果然缺点啥~~（似乎开服解决了抽卡记录难题以后就都摆了？）~~

因为原神那边有椰羊之类的在做扫描，但是HSR的成就太过离谱而且有很多隐藏的，而且还会有成对的，所以很难扫描？

（话说UIGF之前有参考过解包出来的字符串列表吗？（以及总感觉用了解包字符串的话要被DMCA干碎吧））
